package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "preq",
-  "version": "0.5.3",
+  "version": "0.6.0",
```

Comment:
I would classify this PR as a bug fix, so IMHO we should bump the patch version only, so that the clients can get it.